main.py

```diff
@@ -14,8 +14,14 @@
 from pymongo.errors import DuplicateKeyError
 
 from kytos.core import KytosNApp, rest
-from kytos.core.rest_api import (HTTPException, JSONResponse, Request,
-                                 Response, error_msg, get_json_or_400)
+from kytos.core.rest_api import (
```

Comment:
Why the formatting change? Not only in this snippet but throughout all the files you modified?

Reply:
I just used Black

Reply:
I don't know if there's another linter, but its the only one I'm familiar with since Vinicius told me about it